medicines/web/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM node:12
```

Comment:
Please can you bind this version with was define on `.nvmrc` and also to be the [alpine version](https://hub.docker.com/_/node/)
```
FROM node:12.13.0-alpine
```